docs/functional-tests.md

```diff
@@ -13,8 +13,16 @@
 ## Basic VM negative
 | Test description | Implemented |
 | :---------------- | :-----------: |
-| Basic VM import with invalid VM image | &check; |
-| Basic VM import with invalid VM image size | &check; |
+| Basic VM import with invalid VM image should fail import | &check; |
+| Basic VM import with invalid VM image size should fail import | &check; |
```

Comment:
what 'invalid' stands for ? wrong value instead of number or size too big for storage class to provide ?

Reply:
From the looks of the test that was written it's a number that does not correspond to the actual image size (i think). 
@machacekondra - do you confirm that?

Reply:
Right, that's actually the test case that we properly handle when CDI pod keeps crashing, and it end with `crashbackoflopp` state